examples/motoko_examples/hello/canisters/rust/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+#[ic_cdk_macros::query]
+pub fn greet(name: String) -> String {
+    format!("Hello, {name}!")
```

Comment:
This works? I didn't think format! worked like this

Reply:
Yes, I think it's how it does it in the default rust canister